samples/Client/DaprClient/Program.cs

```diff
@@ -3,353 +3,42 @@
+        private static readonly Example[] Examples = new Example[]
         {
-            var builder = new ConfigurationBuilder();
-            builder.AddCommandLine(args);
-
-            var config = builder.Build();
-
-            var jsonOptions = new JsonSerializerOptions()
-            {
-                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
-                PropertyNameCaseInsensitive = true,
-            };
-
-            var client = new DaprClientBuilder()
-                .UseJsonSerializationOptions(jsonOptions)
-                .Build();
-
-            await PublishEventAsync(client);
-
-            // Save State
-            await SaveStateAsync(client);
-
-            // Read State
-            await GetStateAsync(client);
-
-            // Delete State
-            await DeleteStateAsync(client);
-
-            // State Transaction
-            await ExecuteStateTransactionAsync(client);
-
-            // Read State
-            await GetStateAfterTransactionAsync(client);
-
-            // Read and save state with etags
-            await ReadAndSaveStateWithEtagsAsync(client);
-
-            if (config.GetValue("rpc-exception", false))
-            {
-                // Invoke /throwException route on the Controller sample server
-                await InvokeThrowExceptionOperationAsync(client);
-            }
-
-            // Invoke deposit operation on ControllerSample or RoutingSample or GrpcServiceSample service by publishing event.
-            await PublishDepositeEventAsync(client);
-
-            await Task.Delay(TimeSpan.FromSeconds(1));
-
-            #region Service Invoke - Required RoutingService
-            // This provides an example of how to invoke a method on another REST service that is listening on http.
-            // To use it run RoutingService in this solution.
-
-            if (config.GetValue("useRouting", false))
-            {
-                //Invoke deposit operation on RoutingSample service by POST.
-                await InvokeDepositServiceOperationAsync(client);
-
-                //Invoke withdraw operation on RoutingSample service by POST.
-                await InvokeWithdrawServiceOperationAsync(client);
-
-                //Invoke balance operation on RoutingSample service by GET.
-                await InvokeBalanceServiceOperationAsync(client);
-            }
-            #endregion
-
-            #region Service Invoke via GRPC - Required GrpcServiceSample
-
-            if (config.GetValue("useGrpcsample", false))
-            {
-                //Invoke deposit operation on GrpcServiceSample service by GRPC.
-                await InvokeGrpcDepositServiceOperationAsync(client);
-
-                //Invoke withdraw operation on GrpcServiceSample service by GRPC.
-                await InvokeGrpcWithdrawServiceOperationAsync(client);
+            new InvokeServiceGrpcExample(),
+            new InvokeServiceHttpExample(),
```

Comment:
InvokeServiceHttpClientExample is missing

Reply:
Good catch